src/components/StepTitle/index.tsx

```diff
@@ -0,0 +1,75 @@
+import styled from '@emotion/styled'
+import PropTypes from 'prop-types'
+import React, { ComponentProps, ReactNode } from 'react'
+import Bullet from '../Bullet'
+
+export const Step = styled('div', {
+  shouldForwardProp: prop => !['disabled'].includes(prop.toString()),
+})<{ disabled: boolean }>`
+  margin-bottom: 24px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: ${({ theme, disabled }) =>
+    disabled ? theme.colors.neutral.textWeak : theme.colors.neutral.textStrong};
+`
+
+const StyledDiv = styled('div', {
+  shouldForwardProp: prop => !['size'].includes(prop.toString()),
+})<{ size: Sizes }>`
+  flex: 1;
+  margin-left: ${({ theme, size }) =>
+    size === 'medium' ? theme.space['2'] : theme.space['1']};
+  font-size: ${({ size }) => (size === 'medium' ? '24px' : '16px')};
+`
+
+export type Sizes = 'small' | 'medium' | undefined
+
+type ContentProps =
+  | { bulletIcon: ComponentProps<typeof Bullet>['icon']; bulletText?: never }
+  | { bulletIcon?: never; bulletText: string }
+
+type StepTitleProps = {
+  bulletVariant?: ComponentProps<typeof Bullet>['variant']
+  size?: Sizes
+  disabled?: boolean
+  children: ReactNode
+} & ContentProps
+
+const StepTitle = ({
+  bulletText,
+  bulletIcon,
+  bulletVariant,
+  children,
+  size = 'medium',
+  disabled = false,
```

Comment:
I still think we could call prop `variant` instead of `bulletVariant` and pass it to Bullet component like we do for size prop. I believe that variant will always influence Bullet and Title